packages/desktop/views/dashboard/components/toasts/VersionToast.svelte

```diff
@@ -10,17 +10,14 @@
     }
 
     const localeKey = 'views.dashboard.toasts.version'
-
-    function onClick(): void {
-        openUrlInBrowser('https://medium.com/bloom-wallet/unveiling-bloom-early-access-8038d29c5f86')
-    }
 </script>
 
-<SidebarToast
-    color={TOAST_PROPS[APP_STAGE]}
-    header={localize(`popups.appUpdate.${APP_STAGE}`)}
-    body={localize(`${localeKey}.body`)}
-    open
-    dismissable={false}
-    button={APP_STAGE === AppStage.PROD ? { text: localize(`${localeKey}.button`), onClick } : undefined}
-/>
+{#if APP_STAGE !== AppStage.PROD}
+    <SidebarToast
+        color={TOAST_PROPS[APP_STAGE]}
+        header={localize(`popups.appUpdate.${APP_STAGE}`)}
+        body={localize(`${localeKey}.body`)}
```

Comment:
For this 1 location, we dont need a `localeKey` var

```suggestion
        body={localize('views.dashboard.toasts.version.body')}
```